gpytorch/models/exact_prediction_strategies.py

```diff
@@ -141,10 +142,10 @@ def get_fantasy_strategy(self, inputs, targets, full_inputs, full_targets, full_
         small_system_rhs = targets - fant_mean - fant_train_covar.matmul(self.mean_cache)
         # Schur complement of a spd matrix is guaranteed to be positive definite
         if small_system_rhs.requires_grad or schur_complement.requires_grad:
-            # TODO: Delete this part of the if statement when PyTorch implements potrs derivative.
+            # TODO: Delete this part of the if statement when PyTorch implements cholesky_solve derivative.
```

Comment:
Let's make an issue for this and reference it here.

Reply:
Will do